client/x/evmengine/keeper/abci.go

```diff
@@ -230,7 +230,7 @@ func (k *Keeper) PostFinalize(ctx sdk.Context) error {
 	maxWithdrawals, err := k.evmstakingKeeper.MaxWithdrawalPerBlock(ctx)
 	if err != nil {
 		log.Error(ctx, "Starting optimistic build failed; get max withdrawal", err, logAttr)
-		return errors.Wrap(err, "get max withdrawal per block")
+		return nil
```

Comment:
this looks like a behaviour change, is it expected?

Reply:
Good point. In conclusion, yes. Because when error occurs from `PostFinalize`, it is just printed and ignored as it is just for building payload optimistically.